omreader/src/OMReader.scala

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2025 Yuhang Zeng <unlsycn@unlsycn.com>
```

Comment:
Upstream this file to zaozi as user API, and keep your copyright there. In t1 repo all copyright are reserved by me.